src/main/conf/import.json

```diff
@@ -13,19 +13,20 @@
       "key": "delim",
       "title": "Delimiter Character",
       "default": ",",
-      "maximum": 1,
+      "maxLength": 1,
```

Comment:
How would a tab character be provided? If it's `\t`, that makes 2 chars.

Reply:
if no character is provided to delim, it will fallback on read_table wich will read space characters as delimiters.

Reply:
I'm not sure if read_delim should handle that itslef. If so then I can remove read_table.

Reply:
We could use read_table2 instead because it seems to be more lax than read_table.

Reply:
That's not my point. I am just asking how to specify that the delimiter is a tab?

Reply:
no delim character means we use read_table, I changed it to use read_table2 to be more lenient.

Reply:
I am not talking about specifying no delimiter, I am talking about a tabulation separated file.